readme.txt

```diff
@@ -53,6 +53,7 @@ The purpose of this plugin is to make it simple to get targeted feedback from cl
 * Improvement: Allow Guests to Comment setting text changed as Allow Site Visitors to view and add comments.
 * Improvement: Renamed settings menu title from "Feedback" to "ProjectHuddle" and added white label support.
 * Improvement: Connection details input box disabled after connection is established.
+* Improvement: Help link added for Detailed instruction on How to connect the site manually?.
```

Comment:
Never add changelog to already released version.. a new version should be used here